src/Products/urban/migration/update_290.py

```
@@ -283,3 +284,29 @@ def add_architect_folder_view(context):
     architects_folder = portal["urban"]["architects"]
     architects_folder.setLayout("architects_folderview")
     logger.info("upgrade step done!")
+
+
+def add_codt_uniquebordering_licences(context):
```

Ruff (0.15.2)

[warning] Unused function argument: `context`

(ARG001)